handbook/engineering/web/frontend-platform/index.md

```diff
@@ -84,6 +84,15 @@ We plan and track our day-to-day work on our [Kanban board](https://github.com/o
 
 The team makes use of [tracking issues](../../tracking_issues.md) for tracking progress on the implementation of new features. The teammates should ensure that a tracking issue is created when starting work on features that are expected to take longer than a few days (or require multiple PRs) to deliver.
 
+#### Getting cross-team feedback on RFC
+
+- Create a [tracking issue](../../tracking_issues.md) on our [Kanban board](https://github.com/orgs/sourcegraph/projects/144).
+- Create a separate issue for the RFC tracking.
```

Comment:
Is there a way to block comments on issues? 

Issues naturally are spaces for discussions and we are using it as a way to track an external source (Google Docs).

Reply:
Yep, it we can [lock the conversation](https://docs.github.com/en/github/building-a-strong-community/locking-conversations).

Reply:
Added note about that 👍